rules/config

```diff
@@ -224,3 +224,12 @@ ENABLE_ASAN ?= n
 
 # reset default container registry from dockerhub to other
 DEFAULT_CONTAINER_REGISTRY ?=
+
+# ENABLE_FIPS_FEATURE - support FIPS feature, only for amd64 or arm64, armhf not supported yet
+# ENABLE_FIPS - support FIPS flag, if enabled, no additional config requred for the image to support FIPS
+ifeq ($(CONFIGURED_ARCH), armhf)
```

Comment:
>CONFIGURED_ARCH [](http://example.com/codeflow?start=8&length=15)

`CONFIGURED_ARCH` is defined in Makefile.work. Ideally this file should only contain simple configurable variable, not depend on others. If you want to override, you may do in Makefile.work

Reply:
Qi, thanks for comment. Agree, move the logic to the Makefile.work, and keep rule/config simple and clean.

Reply:
Will you remove these line in this file?

Reply:
Removed, thanks.